### DISCUSSION TITLE
Toggle Permissions and Visibility for Discussions in Public Repo

### DISCUSSION THREAD

**Discussion by JackEAllen** [community/community#11123] [Discussions]
I understand private discussion can already take place for a team, but not in a specific repository discussion page.

I would like to be able to create both open and private discussions for an open source repository's discussion page. 
Would it be possible to toggle permissions and visibility for discussions on an open source repository? 

For example, organisation members or members with access to the repository containing the discussion page only, can have private discussions. Private discussion would allow developers to discuss more sensitive information in a discussion on a repositories discussions page, while also allowing more open conversations to be discussed in the same place.  

In other words, I would like to create permission groups and toggle visibility of discussions for a given repositories discussions page based on permission groups. I imagine this could potentially be implemented on creation of a new discussion.

**Comment by kwin** [comment]
This is now more important because team discussions do no longer exist (https://github.blog/changelog/2023-02-08-sunset-notice-team-discussions/)

**Comment by tounsoo** [comment]
I also think we need the reverse as well. We have a private repo but we want feedback from everyone that isn't the developer of the project.
